student.rb

```diff
@@ -11,4 +11,9 @@ def initialize(age, classroom, name: 'Unknown', parent_permission: true)
   def play_hooky
     '¯\\_(ツ)_/¯'.encode('utf-8')
   end
+
+  def set_classroom(classroom) # rubocop:disable Naming/AccessorMethodName
+    @classroom = classroom
+    classroom.students << self
```

Comment:
- [ ] before pushing the student to the classroom you need to check if the student is already in the classroom.